content/en/synthetics/cicd_integrations/_index.md

```diff
@@ -408,311 +66,34 @@ The test triggering endpoint supports starting up to 50 tests in one request.
 }
 ```
 
-The `TEST_TO_TRIGGER` objects are composed of the `public_id` (required) of the test to trigger and optional configuration overrides ([see below](#configure-tests) for a description of each field).
-
-The public identifier of a test can be either the identifier of the test found in the URL of a test details page (for `https://app.datadoghq.com/synthetics/details/abc-def-ghi`, it would be `abc-def-ghi`) or the full URL to the details page (that is, `https://app.datadoghq.com/synthetics/details/abc-def-ghi`).
-
-#### Example request
-
-```bash
-#!/bin/sh
+The `TEST_TO_TRIGGER` objects compose of the required `public_id` for the test you want to trigger and the optional configuration overrides. For descriptions of each field, see [Configure tests](#configure-tests).
 
-api_key="<DATADOG_API_KEY>"
-app_key="<DATADOG_APPLICATION_KEY>"
+A test's public identifier is either the identifier of the test found in the URL of a test's details page (for example: the identifier for `https://app.datadoghq.com/synthetics/details/abc-def-ghi` is `abc-def-ghi`) or the full URL of a test's details page (for example: `https://app.datadoghq.com/synthetics/details/abc-def-ghi`).
 
-curl -X POST \
--H 'Content-Type: application/json' \
--H "DD-API-KEY: ${api_key}" \
--H "DD-APPLICATION-KEY: ${app_key}" \
--d '{
-    "tests": [
-        {
-            "public_id": "abc-def-ghi",
-            "allowInsecureCertificates": true,
-            "basicAuth": { "username": "test", "password": "test" },
-            "body": "{\"fakeContent\":true}",
-            "bodyType": "application/json",
-            "cookies": "name1=value1;name2=value2;",
-            "deviceIds": ["laptop_large"],
-            "followRedirects": true,
-            "headers": { "NEW_HEADER": "NEW VALUE" },
-            "locations": ["aws:us-west-1"],
-            "retry": { "count": 2, "interval": 300 },
-            "startUrl": "http://new.url/",
-            "variables": { "titleVariable": "new value" }
-        }
-    ]
-}' "https://api.{{< region-param key="dd_site" >}}/api/v1/synthetics/tests/trigger/ci"
-```
+For more information, see the [Synthetics API endpoint documentation][1].
 
-#### Example response
+### Get details of batch
 
-```json
-{
-  "batch_id": null,
-  "results": [
-    {
-      "result_id": "0123456789012345678",
-      "public_id": "abc-def-ghi",
-      "location": 30019
-    }
-  ],
-  "triggered_check_ids": [
-    "abc-def-ghi"
-  ],
-  "locations": [
-    {
-      "display_name": "N. California (AWS)",
-      "name": "aws:us-west-1",
-      "region": "Americas",
-      "is_active": true,
-      "is_public": true,
-      "id": 30019
-    }
-  ]
-}
-```
-
-### Poll results endpoint
+The get batch details endpoint [@Beth to add description here]...
```

Comment:
@peculiarism please fill this out!

Reply:
```suggestion
The get batch details endpoint allows you to retrieve the test results of the group of tests (known as a batch) that were triggered within your CI/CD pipeline. You will need to provide the `batch_id` for the relevant CI execution.
```

I'm wondering if we should define batch somewhere else in this doc as well? Let me know what you think @alai97

Reply:
I think defining batch in the endpoint description is good!